### ISSUE TITLE
About the computation of feature contrasive loss

### ISSUE THREAD

**Issue by a21401624** [somijeong/MGUIT#2]
I've read the code and this is indeed a great work!
However, I'm confused about the computation of feature contrasive loss(in paper this name, I think indeed this is Info NCE loss right?). You use torch.exp(F.nll_loss(logits, label)) to compute, but I think maybe F.nll_loss(torch.log(logits), label) is the right implementation.
As Info NCE loss is indeed CE loss, and log_softmax + NLLLoss == CE loss. The logits here is the output of softmax, so we only need another torch.log.
Looking forward to the answer.
